gallery/v2_transforms/helpers.py

```diff
@@ -0,0 +1,33 @@
+import matplotlib.pyplot as plt
+from torchvision.utils import draw_bounding_boxes
+
+
+def plot(imgs):
```

Comment:
I think we need to reduce the boilerplate to a very strict minimum in the example, because they're already complex enough as it is - we don't need noise. 

I opted for the option of creating a new file here. I don't have a super strong opinion on how it should be done, other than the fact that this code shouldn't be part of the example itself.

Hopefully this is non-blocking anyway and we can figure out a long-term solution later.